chaos_genius/core/utils/data_loader.py

```diff
@@ -124,15 +142,23 @@ def _build_date_filter(self) -> List[str]:
         if TIMEZONE in SUPPORTED_TIMEZONES:
             tz_offset_string = SUPPORTED_TIMEZONES[TIMEZONE][-6:]
         else:
-            tz_offset_string = datetime.now(pytz.timezone(TIMEZONE)).strftime("%z")
-            tz_offset_string = tz_offset_string[:3] + ":" + tz_offset_string[3:]
+            tz_offset_string = datetime.now(pytz.timezone(TIMEZONE)).strftime(
```

Comment:
Let's have utils functions in the `datetime_helper.py` for this plus functional test cases for those functions.

Reply:
We're going to replace the function as soon as we remove 3 letter timezones. I don't think we need to create a util function for now and then remove it once it's done.

Reply:
Already in [line 140](https://github.com/chaos-genius/chaos_genius/blob/085b7de7df5d94d724db55c83f98c2a10389d56a/chaos_genius/core/utils/data_loader.py#L140).